README.md

```diff
@@ -1,5 +1,7 @@
 # Minimal Perfect Hash Tables
 
+![NetflixOSS Lifecycle](https://img.shields.io/osslifecycle/indeedeng/mph-table.svg)
```

Comment:
Just `OSS Lifecycle`?

Reply:
I had the same question in another PR. This has the feel of a copy/paste issue. That text gets put into the <img> alt attribute.

Reply:
Updated below, this was a copy/paste artifact.